### ISSUE TITLE
main.py 中w_i 未使用，是忘记 注释还是论文的笔误

### ISSUE THREAD

**Issue by luxiaogen** [BIT-DA/ABS#2]
logits_batch_crop_class = weighted_similarity_matrix.sum(dim=-1) # 对每个类别的描述数量维度求和，得到 (B, crop_num, C) [100, 100, M=37]
                        w_i = (
                            (patch_weights_batch * image_scale)
                            .softmax(-1)
                            .unsqueeze(-1) # (B, crop_num, 1) unused ---> 未使用 patch 的权重，
                        ) # [100,100,1]  每张图像内部各个 patch 的归一化权重（类似注意力权重）
                        logits_batch = (ogits_batch_crop_class).sum(dim=1) # TODO [100,200] 
                        logits_total.append(logits_batch)
